code/modules/mob/living/simple_animal/hostile/regalrat.dm

```diff
@@ -78,13 +79,32 @@
 
 /mob/living/simple_animal/hostile/regalrat/AttackingTarget()
 	. = ..()
+	if(health >= maxHealth)
+		to_chat(src, "<span class='warning'>You feel fine, no need to eat anything!</span>")
+		return
 	if(istype(target, /obj/item/reagent_containers/food/snacks/cheesewedge))
-		if (health >= maxHealth)
-			to_chat(src, "<span class='warning'>You feel fine, no need to eat anything!</span>")
-			return
 		to_chat(src, "<span class='green'>You eat \the [src], restoring some health.</span>")
 		heal_bodypart_damage(10)
```

Comment:
I know this isn't touched by the PR, but since the other numbers should be defines this should be moved into one as well for consistency, and because it's easy